src/commands/app-configs/deploy.ts

```diff
@@ -97,4 +98,16 @@ the application configuration was created.
     const data = result.data as any
     return data.deployments
   }
+
+  // Show a warning if a user tries to deploy an app-config when it is already deployed
+  handleError(error: MixError) {
+    debug('handleError() error.statusCode: %d', error.statusCode)
+
+    if (error.statusCode === 400 && error.message.toLocaleLowerCase().includes('deployment already completed')) {
```

Comment:
The reason why the "Deploying application configuration...  done" message appeared after the warning is because we need to stop the spinner.

A call to `CliUx.ux.action.stop()` is needed prior to outputting the warning here. We should pass the string "Aborted" to the `stop` function and use the chalk library to make that "Aborted" yellow as it is  a warning.